src/Faker/Provider/PhoneNumber.php

```diff
@@ -9,8 +9,8 @@ class PhoneNumber extends \Faker\Provider\Base
     /**
      * @example '555-123-546'
      */
-    public static function phoneNumber()
+    public function phoneNumber()
```

Comment:
This will break many locales overriding this method with a `static` one. If you change it here, you must change it in locales, too.